### PR TITLE
Improve interoperability test suite: Add more RPC methods to TestApi

### PR DESCRIPTION
**Pull request by VojtechVitek**
- Add new RPC methods (test empty, one, multiple request/response arguments)
- Rename to TestApi
- Define fixtures as variables
- Use `cmp.Equal()` and `cmp.Diff()` for payload comparison
- Improve server code for https://github.com/webrpc/gen-golang/pull/14

### CODE REVIEW

**Comment by VojtechVitek** [comment]
```
=== RUN   TestInteroperability
    interoperability_test.go:21: 
        	Error Trace:	/home/runner/work/webrpc/webrpc/tests/interoperability_test.go:21
        	Error:      	Received unexpected error:
        	            	Failed tests:
        	            	SendComplex(): webrpc invalid argument error: "complex":
        	            	  &server.Complex{
        	            	  	Meta: map[string]any{
        	            	  		"1": string("23"),
        	            	- 		"2": float64(24),
        	            	+ 		"2": int(24),
        	            	  	},
        	            	  	MetaNestedExample: {"1": {"2": 1}},
        	            	  	NamesList:         {"John", "Alice", "Jakob"},
        	            	  	... // 7 identical fields
        	            	  }
        	            	
        	Test:       	TestInteroperability
--- FAIL: TestInteroperability (0.01s)
```

File: tests/schema/test.ridl [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+  - GetError()
+  
+  - GetOne() => (one: Simple)
+  - SendOne(one: Simple)
+
+  - GetMulti() => (one: Simple, two: Simple, three: Simple)
+  - SendMulti(one: Simple, two: Simple, three: Simple)
+  
+  - GetComplex() => (complex: Complex)
+  - SendComplex(complex: Complex)
```
Comment (VojtechVitek) [inline tests/schema/test.ridl:17]: This is the updated Interoperability TestApi schema. All generators should run tests against reference webrpc-test server.

File: tests/error_test.go [hunk 2]
```diff
@@ -1,56 +0,0 @@
...
-	if err, ok := err.(client.Error); ok {
-		assert.Equal(t, client.ErrNotFound, err.Code())
-		assert.Equal(t, "resource was not found", err.Msg())
-		err.Msg()
-	} else {
-		assert.Fail(t, "error must be type of client.Error, actual %v", err)
-	}
-}
-
-func TestInvalidArgument(t *testing.T) {
```
Comment (VojtechVitek) [inline tests/error_test.go:42]: I'll want to re-add / revisit these tests when working on custom errors. Removed for now.
